protobuf/lib/src/protobuf/utils.dart

```diff
@@ -35,4 +35,27 @@ bool _areByteDataEqual(ByteData lhs, ByteData rhs) {
   return _areListsEqual(asBytes(lhs), asBytes(rhs));
 }
 
-List<T> sorted<T>(Iterable<T> list) => List.from(list)..sort();
+List<T> sorted<T>(Iterable<T> list) => new List.from(list)..sort();
```

Comment:
Let's deprecate this, and make a private version. It has nothing to do in the exports of protobuf.